api-2.0/pom.xml

```diff
@@ -54,9 +57,76 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.openmrs.module</groupId>
+            <artifactId>emrapi-api</artifactId>
+            <version>${emrapiVersion}</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.openmrs.module</groupId>
+            <artifactId>emrapi-api</artifactId>
+            <version>1.12</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
The reason why the Travis CI build is still failing is because this should be changed to:
```xml
<dependency>
  <groupId>org.openmrs.module</groupId>
  <artifactId>emrapi-api-1.12</artifactId>
  <version>${emrapiVersion}</version>
  <scope>test</scope>
</dependency>
```

Reply:
@mks-d thank you very much for noting that.